integrations/google_genai/src/haystack_integrations/components/generators/google_genai/chat/chat_generator.py

```diff
@@ -89,6 +106,54 @@ def _convert_message_to_google_genai_format(message: ChatMessage) -> types.Conte
                 )
             )
 
+    # Handle images (from user only)
+    if message.images:
+        if message.is_from(ChatRole.ASSISTANT):
+            msg = "Image content is not supported for assistant messages"
+            raise ValueError(msg)
+
+        for image_content in message.images:
```

Comment:
I would prefer to use a for-loop for all parts in `ChatMessage._content`.
This would preserve the order or textual and images part, which can be relevant.

I'm referring to something like
```
Here are some images. Image 1:
[IMAGE 1]
Image 2
[IMAGE 2]
Please describe them
```

(I'm not doing this in Ollama because it only allows passing a list of images instead of a list of multimodal parts)